about.html

```diff
@@ -7,12 +7,5 @@
 
 Here at DevsKrate we inspire and endeavor to share everything about technology like Softwares, Coding, etc., and provide our best about HOW TO tutorials and BEST OF every category in accordance to our knowledge.
 
-<small>
-Octocat is Github's logo and trademark and here is used to illustrate a fictional character (John Smith) that used the *{ Personal }* Jekyll Theme to build his website.
-</small>
```

Comment:
why dont keep this small tags, maybe we wil forget this in future

Reply:
will add again